roles/addons/lvm/readme.rst

```diff
@@ -0,0 +1,160 @@
+LVM
+---
+
+Description
+^^^^^^^^^^^
+
+This role configure local LVM stack: Physical Volumes, Volume Groups, and Logical Volumes.
+
+Role assume standard LVM tree, with *vg* at center of it. For example:
+
+.. code-block:: text
+
+  +---------+
+  |   pv1   +-----+
+  +---------+     |                         +---------+
+                  |                    +--->+   lv1   |
+                  |                    |    +---------+
+  +---------+     |    +---------+     |
+  |   pv2   +--------->+   vg1   +-----+
+  +---------+     |    +---------+     |
+                  |                    |    +---------+
+                  |                    +--->+   lv2   |
+  +---------+     |                         +---------+
+  |   pv3   +-----+
+  +---------+
+
+Instructions
+^^^^^^^^^^^^
+
+All options from lvg and lvol ansible modules are supported. See: 
+
+* lvg: https://docs.ansible.com/ansible/latest/collections/community/general/lvg_module.html
+* lvol: https://docs.ansible.com/ansible/latest/collections/community/general/lvol_module.html
+
+To add LVM to an host, use the following YAML structure:
+
+.. code-block:: yaml
+
+      hosts:
+        management1:
+          network_interfaces:
+            - interface: enp0s8
+              ip4: 10.10.0.1/16,10.10.0.2/16
+              mac: 08:00:27:36:c0:ac
+              network: ice1-1
+          storage:
+            lvm:
+              - vg: data
```

Comment:
Why do you tie vg and lv together in this structure?
I would rather use 2 parameters, one to define vg (to use with lvg) and the other to define lv (to use with lvol).
This could allow to define and inherit a single VG for hosts that share the same storage topology, but with different needs for logical volumes.

Reply:
Not sure I understand your need.
It is highly possible I missed something when creating this structure.

Could you provide a small example ?

Reply:
Correcting, should be ok in next commit.